### ISSUE TITLE
tput: unknown terminfo capability 'AF'

### ISSUE THREAD

**Issue by blshkv** [drwetter/testssl.sh#1288]
There are few such unnecessary (and harmless) lines at the very beginning of the output. The latest version rc5 affected only. It weren't any in the previous version (2.9)

1. uname -a
Linux pt 5.1.15-pentoo #1 SMP PREEMPT Sat Jun 29 11:07:18 +08 2019 x86_64 Intel(R) Core(TM) i5-3320M CPU @ 2.60GHz GenuineIntel GNU/Linux

2. testssl version from the banner: testssl.sh -b 2>/dev/null | head -4 | tail -2
3.0rc5 from https://testssl.sh/dev/

4. openssl version used by testssl.sh: testssl.sh -b 2>/dev/null | awk -F':' '/openssl/ { print $2}'
dev-libs/openssl-1.1.1c
dev-libs/openssl-bad-1.0.2_p20190119

5. steps to reproduce: testssl.sh or docker command line, if possible incl. host
just run the tool

6. what exactly was happening, output is needed
```
bash$ testssl.sh --html traveler.uobkayhian.com
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'AF'
tput: unknown terminfo capability 'md'
tput: unknown terminfo capability 'md'
tput: unknown terminfo capability 'us'
tput: unknown terminfo capability 'mr'
tput: unknown terminfo capability 'me'

###########################################################
    testssl.sh       3.0rc5 from https://testssl.sh/dev/

      This program is free software. Distribution and
             modification under GPLv2 permitted.
      USAGE w/o ANY WARRANTY. USE IT AT YOUR OWN RISK!

       Please file bugs @ https://testssl.sh/bugs/

###########################################################

 Using "OpenSSL 1.0.2-chacha (1.0.2k-dev)" [~197 ciphers]
 on pt:/usr/bin/openssl-bad
 (built: "Apr 24 02:33:53 2019", platform: "linux-x86_64")


 Start 2019-07-03 14:43:07
```

```
env | grep TERM
COLORTERM=truecolor
TERM=xterm-color
```

**Comment by drwetter** [comment]
Thanks for reporting!

Found the culprit. Stay tuned

**Comment by drwetter** [comment]
closed via PR #1289
